src/main/pg/mco.h

```diff
@@ -27,7 +27,13 @@
 #include "drivers/io_types.h"
 
 typedef struct mcoConfig_s {
+#ifdef STM32G4
```

Comment:
Parameter groups cannot contain conditionals, as this would make their format inconsistent - just make the entries for these parameters in `settings.c` conditional.

Reply:
Ah yeah... It's okay to contain conditionals in reset templates/functions, right?

Reply:
Yes.